### PR TITLE
[4.0] remove adddsubmenu

### PR DESCRIPTION
**Pull request by brianteeman**
Removes the last remnants of the addSubmenu code which is no longer used.

NOTE I have left the com_finder component out of this PR as it is already addressed elsewhere in a PR awaiting merge

### CODE REVIEW

**Comment by richard67** [comment]
> NOTE I have left the com_finder component out of this PR as it is already addressed elsewhere in a PR awaiting merge

Other PR for com_finder is #27032 .

File: administrator/components/com_workflow/Helper/WorkflowHelper.php [hunk 1]
```diff
@@ -24,38 +24,5 @@
...
-	 *
-	 * @param   string  $vName  The name of the active view.
-	 *
-	 * @return  void
-	 *
-	 * @since   3.1
-	 */
-	public static function addSubmenu($vName)
-	{
-		$extension = Factory::getApplication()->input->getCmd('extension');
```
Comment (Quy) [inline administrator/components/com_workflow/Helper/WorkflowHelper.php:38]: Remove obsolete `use` statements. Check other files if applicable.

Comment (brianteeman) [reply administrator/components/com_workflow/Helper/WorkflowHelper.php:38]: done

File: administrator/components/com_fields/Helper/FieldsHelper.php [hunk 2]
```diff
@@ -678,56 +678,6 @@ public static function getFieldsPluginId()
...
-
-		// Avoid nonsense situation.
-		if ($component == 'com_fields')
-		{
-			return;
-		}
-
-		// Try to find the component helper.
-		$eName = str_replace('com_', '', $component);
-		$file  = Path::clean(JPATH_ADMINISTRATOR . '/components/' . $component . '/helpers/' . $eName . '.php');
```
Comment (Quy) [inline administrator/components/com_fields/Helper/FieldsHelper.php:710]: Remove `use Joomla\CMS\Filesystem\Path;`.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 12b3fa1eef7ff6e79f015bfeeb888704c022a79e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27381">issues.joomla.org/tracker/joomla-cms/27381</a>.</sub>

**Comment by HLeithner** [comment]
Thanks

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**